src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.objects.Ball;
+import java.util.Random;
+
+public class Lottery {
+    private final Random random;
+    private final int maxNumberOnTheBall;
```

Comment:
It can be constant in this case

Reply:
What can be constant in this case ? Two fields are already final. Class is not used in inheritance, so there is no need for final class.

Reply:
Not understand what 'It can be constant in this case'. Fields are already final, should they be public static ?